### DISCUSSION TITLE
Welcome to shredos.x86_64 Discussions!

### DISCUSSION THREAD

**Discussion by PartialVolume** [PartialVolume/shredos.x86_64#57] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sciaud1** [comment]
Which Linux distro was used as a base for the program? Thanks

**Reply by PartialVolume** [reply]
It's built using `buildroot` which is a make file so not really a Linux distro in the usual sense. Buildroot is predominantly used for creating embedded systems on a chip where you only include software components specific to the hardware. That results in a very small OS ideal for embedded applications.

ShredOS pulls a lot more software into its build than an embedded system, however we still use buildroot simply because that's what DBAN used and there are plenty of other distros that include nwipe in their repositories.

**Reply by sciaud1** [reply]
thank you very much indeed. For a personal project I would like to create a data wiping program that can be started from Windows BIOS. I would like to write the data wiping software in Python. But I wanted to start from a Linux base. Do you think buildroot is suitable for my project? Thanks.

**Reply by PartialVolume** [reply]
If by Windows BIOS you mean the UEFI, then maybe, although whether you can write a UEFI in python I don't know. I think C is usually the language used?

How compatible it would be across platforms is another question. ShredOS uses many Linux  kernel drivers to make it compatible with many different graphics cards, keyboards, USB bridges, processors which UEFI won't have access to as the OS hasn't booted at that stage.

I could be being a bit overly negative here, but I would have thought it would be quite a challenge! But then again I've never written code for the UEFI, so can't really talk knowledgeably about this.

**Reply by sciaud1** [reply]
Thank you so much. Maybe use a Linux distro like ubuntu server for my project is better. I don't know. However good job and Thank you
